packages/shared/styles/components/SfProductCardHorizontal.scss

```diff
@@ -0,0 +1,124 @@
+@import '../variables';
+
+:root {
+  --product-card-horizontal-padding-content: var(--spacer-small);
+  --product-card-horizontal-padding: var(--spacer-medium);
+  --product-card-horizontal-padding-desktop: var(--spacer-big);
+  --product-card-horizontal__image-max-width: 85px;
+  --product-card-horizontal__image-max-width-desktop: 140px;
+  --product-card-horizontal__price-font-weight: 500;
+}
+
+.sf-product-card-horizontal {
+  $this: &;
+  box-sizing: border-box;
+  position: relative;
+  display: flex;
+  padding: var(--product-card-horizontal-padding-content);
+  &:hover {
+    #{$this}__picture {
+        opacity: 0;
+        &:nth-child(even){
+          opacity: 1;
+        }
+      }
+  }
+  &__image-wrapper {
```

Comment:
@martaradziszewska, please check image styles because we add some improvements to SfImage 
1. remove options from &__image-wrapper and &__image/&__picter
2. add just necessary styles
check SfProductCard styles it's should be similar